modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -106,6 +110,14 @@ The following snippet sets DNS configuration on the host.
         - 8.8.8.8
 ...
 ----
+<1> You must configure an interface when configuring the DNS resolver. A manifest that changes the DNS resolver configuration and does not include an interface configuration will not work.
```

Comment:
I think I would combine these two points into something like:

> You must configure an interface with `auto-dns: false` or you must use static IP configuration on an interface in order for Kubernetes-NMState to store custom DNS settings.

Strictly speaking, if you had an interface with a static IP already you wouldn't need the interface section in the DNS policy. You just need _some_ interface that can store static DNS settings.